src/config.ts

```diff
@@ -23,6 +23,7 @@ export interface RenderingConfig {
   verboseLogging: boolean;
   dumpio: boolean;
   timingMetrics: boolean;
+  headed?: boolean;
```

Comment:
I think it would be nice to add this in the `default.json` and `dev.json` files set to false so users and developers know that it exists.